languages/julia/exercises/concept/functions-introduction/.docs/after.md

```diff
@@ -0,0 +1,42 @@
+### Summary
+
+After solving this exercise, you should be familiar with:
```

Comment:
This might be very subtle, but maybe rephrase a little bit:

```suggestion
This exercise showed how to:
```

Reply:
That sounds a lot better, thanks!